RHDEVDOCS-3750+3704 Tekton Hub with OpenShift Pipelines (#43307)
openshift/openshift-docs: modules/op-deploying-tekton-hub.adoc

// This module is included in the following assembly:
//
// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc

:_content-type: PROCEDURE
[id="deploying-tekton-hub_{context}"]
= Deploying {tekton-hub}

[role="_abstract"]
Cluster administrators can deploy {tekton-hub} on their enterprise infrastructure. 

[NOTE]
====

> If you are using Github Enterprise or Gitlab Enterprise, deploy Hub in the same network where the enterprise server is running. For example, if enterprise server is running behind VPN, then the Openshift Cluster on which you are deploying {tekton-hub} must also be behind VPN.

Reviewer: Check capitalization of GitHub and GitLab throughout. 
Check that we use the product name, GitLab Enterprise Edition, consistently througout.

Reviewer: ```suggestion
If you are using GitHub Enterprise or GitLab Enterprise Edition, deploy {tekton-hub} in the network where the enterprise server is running. For example, if the enterprise server is behind a VPN, the OpenShift cluster where you deploy {tekton-hub} must also be behind the VPN.
```
